rfcs/20180507-cond-v2.md

```diff
@@ -0,0 +1,170 @@
+# **"Functional"** **cond design doc**
+
+| Status        | In-progress       |
```

Comment:
This is an accepted design doc we're retrospectively publishing, so should the status be Approved?

Reply:
Sure. I put in-progress since we're still working on the implementation, but it's true that the design itself is approved. I'll change it.

Reply:
Should be fixed now.